BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -319,7 +319,7 @@ extension AutoContributeDetailViewController: UITableViewDataSource, UITableView
       cell.selectionStyle = .none
       
       if let url = URL(string: publisher.url) {
-        state.dataSource?.retrieveFavicon(with: url) { data in
+        state.dataSource?.retrieveFavicon(for: url, faviconURL: URL(string: publisher.faviconUrl)) { data in
```

Comment:
can `faviconUrl` be set to `clear` string here? or will it be nil when `clear` param is passed?
I remember desktop doing having that `clear` property

Reply:
`clear` is no longer an option here, it'll just be nil. `clear` is taken care of in the `-[BATLedgerDatabase insertOrUpdatePublisherInfo:completion]` call and replaced with an empty string (which will then become nil here)

Reply:
As seen here: https://github.com/brave/brave-core/pull/2866/files#diff-22cd939021706f39a7c4ec71cb224917R154